### PR TITLE
fix: minor formatting change

### PR DESCRIPTION
**Pull request by isindir**

### CODE REVIEW

File: helmenv.sh [hunk 1]
```diff
@@ -7,8 +7,8 @@ function _helmenv_test_requirements {
         exit 1
     elif [[ ! "$(command -v jq)" ]]
     then
-         echo "You must install jq"
-         exit 1
+        echo "You must install jq"
+        exit 1
```
Comment (edwinyoung) [inline helmenv.sh:11]: Formatting changes should be in a separate PR.

File: helmenv.sh [hunk 2]
```diff
@@ -154,6 +154,10 @@ function helmenv_uninstall(){
 function helmenv_list(){
     installed_versions="$(find "${HELM_BINARY_PATH}"/ -name '*helm-*' -printf '%f\n' | grep -Eo 'v([0-9]\.?)+$' | sed '/^$/d' | sort --version-sort)"
     echo "$installed_versions"
+    if [[ -n $installed_versions ]]; then
+        echo
+        helm version -c
+    fi
```
Comment (edwinyoung) [inline helmenv.sh]: This breaks the `fzf` functionality in [`helmenv_use`](https://github.com/alexppg/helmenv/pull/11/files#diff-40a8084ede4c21c311ad91386dbbe9b7R167), since it uses the output of `helmenv_list` to populate the list of helm versions to choose from for `fzf`.

**Review by edwinyoung — COMMENTED** [review]
Current changes are breaking - perhaps there should be a secondary function created to be used with `fzf` in the event that no helm version is supplied to the `helmenv use` command?

File: helmenv.sh (hunk 2 above)
Comment (isindir) [reply helmenv.sh]: ok, it does not make sense then to add this feature, as I can use `helm version -c` standalone

Comment (alexppg) [reply helmenv.sh]: I agree, it's not very useful to have a separated command to see the version. But I was thinking about adding an asterisk in the used version in `helmenv_list`.
